### PR TITLE
Update 2020 speakers

### PR DESCRIPTION
**Pull request by glasnt**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -146,175 +108,158 @@ <h2 class="section-title">Speakers</h2>
...
             <div class="img-container">
-              <img src="./images/speakers/tom-eastman.jpg" alt="" />
+              <img src="./images/speakers/katie-mclaughlin.jpg" alt="Katie McLaughlin" />
             </div>
             <div class="details">
-              <span class="name">Tom Eastman</span>
-              <a href="https://twitter.com/tveastman" target="_blank" class="social-handle">tveastman</a>
-              <span class="meta">Using Python, Django, and ruthlessness to protect people from social media harassment.</span>
+              <span class="name">
+                Markus Holterman
```
Comment (MarkusH) [inline index.html]: I think you deserve to have your own name here

Comment (glasnt) [reply index.html]: You sure? Ok.

File: index.html [hunk 2]
```diff
@@ -146,175 +108,158 @@ <h2 class="section-title">Speakers</h2>
...
 
           <li class="option group">
             <span class="name">PyConline AU Contributor</span>
-            <p class="description">Give a little extra to help with financial aid for other attendees. Includes DjangoCon AU.</p>
-            <span class="price"><span class="currency">AU&#36;</span>990</span>
+            <p class="description">All funds raised by contributor tickets are used to support outreach and diversity in future years.</p>
+            <span class="price"><span class="currency">AU&#36;</span>300+</span>
           </li>
 
         </ul>
```
Comment (MarkusH) [inline index.html:265]: The link below in line 268 is wrong: Needs to be https://2020.pycon.org.au/attend/ or https://pretix.eu/pyconau/online2020/

**Review by MarkusH — APPROVED** [review]
One more tiny change. The rest looks great. :rocket:

**Pull request merged**